lab/lab1/part3-data-transformation.html

```diff
@@ -63,15 +63,25 @@
 
     ===================== */
 
-    var data = bikeArrayClean;
+
+
+
 
     /* =====================
 
     Start code to filter data
 
     ===================== */
 
+    var data = bikeArrayClean;
```

Comment:
The challenging part about lab1/part3 is cleaning the dirty data. 

You'll need to at least attempt that to get a green light on this assignment

Reply:
I'm so sorry I forgot this the first time! I don't know how I missed it. Anyway, I believe I just updated the file! First time trying that. Please let me know if that works!